cloudflare/wrangler2
# Add action to publish JSON schema for Wrangler

What this PR solves / how to test:

This adds a GitHub action to generate and publish a JSON schema for wrangler PRs. The next step will be to add it to schemastore.org for automatic inclusion in e.g. VSCode. 

How to test:
- Create a new JSON file and open it in VSCode (this can also work with TOML (i.e. wrangler.toml), but I haven't been able to figure this out in VSCode yet)
- Add the key `"$schema": "..."`
- Verify that VSCode will complain if you try to add properties or values that don't conform to the shape of a Wrangler configuration file.

Associated docs issues/PR:

- TBD

Author has included the following, where applicable:


Reviewer has performed the following, where applicable:

- [ ] Checked for inclusion of relevant tests
- [ ] Checked for inclusion of a relevant changeset
- [ ] Checked for creation of associated docs updates
- [ ] Manually pulled down the changes and spot-tested

Fixes #2376
